src/test/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticatorTest.java

```diff
@@ -906,6 +906,52 @@ private RestRequest buildTokenExchangeRestRequest(
             .withHeaders(ImmutableMap.of("Content-Type", "application/json"))
             .build();
     }
+    @Test
+    public void testJwtWithBackslashHInUsername() throws Exception {
+        String[] testUsernames = {
```

Comment:
Could you add a test with failure scenario as well? `\h`

Reply:
Will do!